### DISCUSSION TITLE
Welcome to livehd Discussions!

### DISCUSSION THREAD

**Discussion by renau** [masc-ucsc/livehd#193] [General]
## 👋 Welcome!
  We just enabled github dicussions for LiveHD. The idea is to share:
 * Ideas/questions/suggestions on how to use the flow (RTL developer)
 * Ideas/questions/suggestions on how to extend the flow (EDA developer) 

If you have a bug under some conditions, the idea is to fill a "issue", if you have an idea/question/suggestion then you do a discussion.

**Comment by shmdez** [comment]
Hi @renau, 
I cannot find the installation guide. 
Where can I find installation and some examples for livehd?
Thank you
